lib/amaysim_logger.rb

```diff
@@ -88,7 +88,7 @@ def create_hash_log_params(log_msg, log_level)
         e = log_msg[:exception]
         log_msg.delete(:exception)
         log_exception(e, log_msg)
-        log_msg[:exception_backtrace] = e.backtrace.join('\n')
+        log_msg[:exception_backtrace] = e.backtrace
```

Comment:
Can you please check that the logs are still readable in sumo logic? The new lines may have been put in to fix the back trace